bc_obps/registration/tests/endpoints/v2/_operations/_operation_id/_registration/test_operation_representative.py

```diff
@@ -171,3 +171,73 @@ def test_register_operation_operation_representative_endpoint_bad_data(self):
 
         # Assert
         assert response.status_code == 422
+
+
+class TestOperationRepresentativePutEndpoint(CommonTestSetup):
```

Comment:
just wrote :musical_note: all these tests :partying_face: we don't need anymore :notes: cause Sepehr did the beautiful permissions tests :partying_face: